module/index.js

```diff
@@ -108,7 +108,7 @@ module.exports = {
   exportApplications: JHipsterApplicationExporter.exportApplications,
   exportApplication: JHipsterApplicationExporter.exportApplication,
   /* JDL exporting */
-  exportToJDL: JDLExporter.exportToJDL,
+  JDLExporter,
```

Comment:
note: this is a breaking change

Reply:
exportToJDL does not seems to be used inside the generator-jhipster but is it used somewhere else ?

Reply:
Maybe. The point is that everything in that file _might_ be used somewhere else (a project having JCore as dependency)